README.md

```diff
@@ -16,6 +16,16 @@
 
 ## Quick Start
 
+Before running LEGO-WEBAPP, you have to compile LEGO-BRICKS
```

Comment:
```suggestion
Before running LEGO-WEBAPP, you have to compile LEGO-BRICKS.
```
Perhaps we should also describe what lego-bricks actually is🧐

Reply:
Updated with a little bit more of a description now